app/javascript/components/map-v2/components/analysis/components/draw-analysis/component.jsx

```diff
@@ -166,11 +167,13 @@ class DrawAnalysis extends PureComponent {
                 </ul>
                 {showWidgets && <Widgets simple analysis />}
                 <div className="disclaimers">
-                  <p>
-                    This algorithm approximates the results by sampling the
-                    selected area. Results are more accurate at closer zoom
-                    levels.
-                  </p>
+                  {zoomLevel < 11 && (
```

Comment:
Just for consistency with the rest of the components, could we call this just `zoom`?

Reply:
I thought about it, but as zoom can also be a verb (function) I ended up with zoomLevel, which I considered to be more specific. But I can change it back to zoom if you prefer it!

Reply:
I am happy with that, and you are correct it is a better name. We can leave it like this.